src/Wallabag/CoreBundle/Resources/views/themes/baggy/Entry/entries.html.twig

```diff
@@ -46,7 +46,7 @@
         <div id="entry-{{ entry.id|e }}" class="{% if listMode == 0 %}entry{% else %}listmode entry{% endif %}">
             <h2><a href="{{ path('view', { 'id': entry.id }) }}" title="{{ entry.title|e|raw }}">{{ entry.title | striptags | truncate(80, true, '…') | default('entry.default_title'|trans) | raw }}</a></h2>
 
-            {% set readingTime = entry.readingTime / app.user.config.readingSpeed %}
+            {% set readingTime = entry.readingTime / app.user.config.readingSpeed * 200 %}
```

Comment:
No need for this anymore, right ?

Reply:
Yes we need it.
- readingTime = 5 (minutes)
- readingSpeed = 300 (WPM)
- readingTime/readingSpeed = 0,016666667 (minutes)
- readingTime/readingSpeed*200 = 3,333333333 (minutes)